src/components/GameBoard.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from "react";
+import Cell from "./Cell";
+
+const width = 7, height = 6;
+
+export default () => {
+  const [isFinished, setIsFinished] = useState<boolean>(false);
+  const [winner, setWinner] = useState<string>("");
+  const [turn, setTurn] = useState<string>("Red");
```

Comment:
Since the available value for the turn is only `Red` and `Yellow`, it should be defined as enum type.